### ISSUE TITLE
ImportError: No module named cross_validation

### ISSUE THREAD

**Issue by Amanda2018genetics** [MicrosoftResearch/Azimuth#30]
Hi Author,
I am new to Python, so I know the problem is   relate to the renaming and deprecation of cross_validation sub-module to model_selection. 
But I don't know how can I fixed.
Look forward to your reply.
Yao

**Comment by fschwach** [comment]
I am running into the exact same problem. The only workaround I can see at the moment is to install sklearn version 0.16 or 0.17 which still have the cross_validation method. As far as I can see in the sklearn docs, it is not just a case of changing the name of the method in the azimuth code but the interface has changed too. 
I also installed an older version of numpy into the virtualenv for azimuth: 1.12.1 because the latest version shows a deprecation warning "DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release." when used with the older sklearn module.
So in summary I installed this into my virtualenv:
```
$ pip install 'scikit-learn==0.17.1'
$ pip install 'numpy==1.12.1'
$ pip install azimuth
```

which works for now but of course it isn't great to be using outdated dependencies.
